presto-hive/src/main/java/com/facebook/presto/hive/parquet/reader/ParquetColumnReader.java

```diff
@@ -194,6 +177,76 @@ public Block readPrimitive(Type type)
         return blockBuilder.build();
     }
 
+    private void readGroup(BlockBuilder blockBuilder, int valueNumber, Type type, List<Integer> lengths)
+    {
+        definitionLevel = definitionReader.readLevel();
+        repetitionLevel = repetitionReader.readLevel();
+        int valueCount = 0;
+        for (int i = 0; i < valueNumber; i++) {
+            do {
+                readValues(blockBuilder, type);
+                try {
+                    valueCount = valueCount + 1;
+                    repetitionLevel = repetitionReader.readLevel();
+                    if (repetitionLevel == 0) {
+                        lengths.add(valueCount);
+                        valueCount = 0;
+                        if (i == valueNumber - 1) {
+                            return;
+                        }
+                    }
+                    definitionLevel = definitionReader.readLevel();
+                }
+                catch (Exception e) {
```

Comment:
so  it is an "expected" exception that you want to ignore?  If so, renamed `e` to `expected`.  Also, can the catch type be narrower?  If not, you will need to handle `InterruptedException` and `PrestoException` specially. Also, add a comment explaining exactly why this happens and why it is acceptable.